x11-libs/cmrt/cmrt-1.0.6.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Intel C for Media RunTime GPU kernel manager"
+HOMEPAGE="https://github.com/intel/cmrt"
+SRC_URI="https://github.com/intel/cmrt/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+DEPEND="
```

Comment:
I did now performed a test emerge locally. This package uses ```pkgconfig, checking pkg-config is at least version 0.9.0... yes```. You need to add ```virtual/pkgconfig``` into ```BDEPEND```. Otherwise, lgtm.